### DISCUSSION TITLE
Where is the list of all capture group names supported by nvim-treesitter for highlighting?

### DISCUSSION THREAD

**Discussion by ahelwer** [nvim-treesitter/nvim-treesitter#1633] [Q&A] [answered]
Stuff like `@keyword`, `@number`, `@comment`, etc. Is there a list anywhere of all of these names and what constructs they're intended to refer to? Maybe this is more a function of what is commonly supported by colorschemes, but I assume there is a standard set. Apologies if this is obvious, I've read the repo's top-level README and looked through this repo's wiki but cannot find it.

**Comment by Joakker** [comment, marked as answer]
[Here](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md)

**Reply by ahelwer** [reply]
Ah, so it was obvious. Thanks!

**Reply by Joakker** [reply]
NP. Took me a bit to find it too 😅
